web/app/js/components/DeploymentDetail.jsx

```diff
@@ -51,7 +50,7 @@ export default class DeploymentDetail extends React.Component {
       pollingInterval: 10000,
       deploy: deployment,
       metrics: [],
-      pods: [],
+      // pods: [],
```

Comment:
I would just delete this rather than commenting it out. 
we're now at the point where we're removing so much that I think it'll just be clutter if we continue to comment things out.

Reply:
Yeah, I meant to comment it out while testing the change, and then delete everything that was commented out, but I think I missed that one, thanks.